sdk/eventhubs/azure_messaging_eventhubs_checkpointstore_blob/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "azure_messaging_eventhubs_checkpointstore_blob"
+version = "0.1.0"
+description = "Azure Event Hubs checkpoint store implementation using Azure Blob Storage"
+readme = "README.md"
+license.workspace = true
+repository.workspace = true
+homepage = "https://github.com/azure/azure-sdk-for-rust"
+authors.workspace = true
+keywords = ["sdk", "azure", "messaging", "cloud", "eventhubs"]
+categories = ["api-bindings"]
+documentation = "https://docs.rs/azure_messaging_eventhubs_checkpointstore_blob"
+edition.workspace = true
+rust-version.workspace = true
+
+[dependencies]
+async-trait.workspace = true
+azure_core.workspace = true
+azure_messaging_eventhubs.workspace = true
+azure_storage_blob.workspace = true
+futures.workspace = true
+serde = { workspace = true, features = ["derive"] }
+serde_json.workspace = true
+time = { workspace = true, features = ["serde"] }
+tracing = { workspace = true }
+uuid = { workspace = true, features = ["v4", "serde"] }
+
+[dev-dependencies]
+azure_core_test = { workspace = true, features = ["tracing"] }
+azure_identity.workspace = true
+tokio.workspace = true
+tracing-subscriber.workspace = true
+
+[features]
+default = []
+
+[[example]]
+name = "checkpoint_store_basic"
+path = "examples/checkpoint_store_basic.rs"
+
+[[example]]
+name = "processor_with_blob_checkpoints"
+path = "examples/processor_with_blob_checkpoints.rs"
```

Comment:
Neither of these should be necessary since the name and the path match. This is usually needed when you need to require crate features or turn on other flags. Doesn't hurt, but just FYI it's unnecessary otherwise. Just more maintenance overhead.